department-of-veterans-affairs/vets-website
# Add radio button focus support

## Summary
In a list loop pattern, when the [Add another ... ] button is pressed, the keyboard focus should be the first focusable element, including a radio input field.

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Sign in via mocked authentication at `http://localhost:3001/sign-in/mocked-auth`
3. Go to the 'Gross monthly income' page (/receives-income)
4. Select Yes
5. On the 'Income sources' page, verify that when another income source is added, the radio input field is focused

## Related issue(s)
[Issue in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73299)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots
n/a

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Issue in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73299)

### Quality Assurance & Testing

### Error Handling


### Authentication
